src/devices/heiman.ts

```diff
@@ -498,6 +562,24 @@ export const definitions: DefinitionWithExtend[] = [
         },
         exposes: [e.water_leak(), e.battery_low(), e.tamper(), e.battery()],
     },
+    {
+        zigbeeModel: ["WaterSensor2-EF-3.0"],
+        model: "HS2WL",
+        vendor: "Heiman",
+        description: "Water leakage sensor",
+        fromZigbee: [fz.ias_water_leak_alarm_1, fzLocal.diagnosticHeiman],
+        toZigbee: [],
+        extend: [m.battery(), m.temperature()],
```

Comment:
```suggestion
        extend: [m.battery(), m.temperature(), m.iasZoneAlarm({zoneType: "water_leak", zoneAttributes: ["alarm_1"]})],
```

All of the toZigbee/fromZigbee/exposes and configure can be removed.